src/WebJobs.Script/Host/BlobLeaseManager.cs

```diff
@@ -202,48 +191,9 @@ private void ProcessLeaseError(string reason)
             }
         }
 
-        private static async Task<ICloudBlob> GetLockBlobAsync(string accountConnectionString, string blobName)
-        {
-            CloudStorageAccount account = CloudStorageAccount.Parse(accountConnectionString);
-            CloudBlobClient client = account.CreateCloudBlobClient();
-
-            return await GetLockBlobAsync(client, blobName);
-        }
-
-        private static async Task<ICloudBlob> GetLockBlobAsync(CloudBlobClient client, string blobName)
-        {
-            var container = client.GetContainerReference(HostContainerName);
-
-            try
-            {
-                await container.CreateIfNotExistsAsync();
-            }
-            catch (StorageException exc)
-            when (exc.RequestInformation.HttpStatusCode == 409 && string.Compare("ContainerBeingDeleted", exc.RequestInformation.ExtendedErrorInformation?.ErrorCode) == 0)
-            {
-                throw new StorageException("The host container is pending deletion and currently inaccessible.");
-            }
-
-            CloudBlockBlob blob = container.GetBlockBlobReference(blobName);
-            if (!await blob.ExistsAsync())
-            {
-                try
-                {
-                    await blob.UploadFromStreamAsync(new MemoryStream());
-                }
-                catch (StorageException exc)
-                when (exc.RequestInformation.HttpStatusCode == 412 || exc.RequestInformation.HttpStatusCode == 409)
-                {
-                    // The blob already exists or a lease has already been acquired.
-                }
-            }
-
-            return blob;
-        }
-
         private void ResetLease()
         {
-            LeaseId = null;
+            this.LockHandle = null;
```

Comment:
Can we add a Style cop error for that?

---
In reply to: [123673112](https://github.com/Azure/azure-webjobs-sdk-script/pull/1615#discussion_r123673112) [](ancestors = 123673112)